.github/workflows/test-build.yaml

```diff
@@ -23,25 +23,17 @@ jobs:
           --health-retries 5
 
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Build test image
-        uses: docker/build-push-action@v2
+      - uses: actions/checkout@v3
+      - name: Install poetry
+        run: pipx install poetry
+      - uses: actions/setup-python@v4
         with:
-          context: .
-          file: docker/Dockerfile
-          push: false
-          target: "development"
-          tags: ghcr.io/${{ github.repository }}:${{ github.sha }}
-
+          python-version: "3.10"
```

Comment:
will dependabot notice and be able to update this python-version; or is that something for us to keep in mind?

Reply:
I think this is something we'll just have to remember, unfortunately. But it would definitely be nice if we could keep these things in sync. Maybe there's a way 🤔